less/index.less

```diff
@@ -1 +1,8 @@
 // Follow the order in the readme
+@import 'variables.less';
+@import 'mixins.less';
+@import 'reset.less';
+@import 'global.less';
+@import 'navigation.less';
+@import 'footer.less';
```

Comment:
I'd put footer on the bottom since it's the last of the page. Just a personal choice though.